journal/updated/07-chapter-6.adoc

```diff
@@ -483,13 +487,13 @@ I should be glad if I could say anything that would be of service to you,
 but you well know that the best help is in yourself.
 O look there--ask of Him who '`gives liberally and upbraids not.`' Neglect
```

Comment:
lets change this quote to the NKJV: ask of Him who 'gives to all liberally and without reproach."